guidance/models/_model.py

```diff
@@ -1019,6 +986,8 @@ def __add__(self, value):
         value : guidance grammar
             The grammar used to extend the current model.
         """
+        # Import in function to guard against circular import
+        from ..library._role import RoleBlock
```

Comment:
I have a feeling that some may object to the import inside of a function... Is anyone else better at resolving issues with circular imports?

Reply:
Well, we haven't tried turning on `ruff` or `flake8` yet.....